diff --git a/CMakeLists.txt b/CMakeLists.txt
index d00a4f9f..b36532e6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -54,7 +54,7 @@ if(WIN32)
   endif()
 endif()
 
-option(BUILD_SHARED_LIBS "Build shared libraries" ON)
+set(BUILD_SHARED_LIBS ON) # Force build libzip as a shared library for Alusus.
 option(LIBZIP_DO_INSTALL "Install libzip and the related files" ON)
 
 option(SHARED_LIB_VERSIONNING "Add SO version in .so build" ON)
